test/sagemaker_tests/huggingface/inference/conftest.py

```diff
@@ -30,12 +30,12 @@
 from .utils import image_utils, get_ecr_registry
 
 logger = logging.getLogger(__name__)
-logging.getLogger('boto').setLevel(logging.INFO)
-logging.getLogger('boto3').setLevel(logging.INFO)
-logging.getLogger('botocore').setLevel(logging.INFO)
-logging.getLogger('factory.py').setLevel(logging.INFO)
-logging.getLogger('auth.py').setLevel(logging.INFO)
-logging.getLogger('connectionpool.py').setLevel(logging.INFO)
+logging.getLogger("boto").setLevel(logging.INFO)
```

Comment:
There are a lot of lint fixes. It may be good to have these fixes as a part of separate PR in the future.

Reply:
Now that they are reviewed, let's go ahead and merge them.